packages/grid/_modules_/grid/components/icons/index.tsx

```diff
@@ -11,4 +11,46 @@ export const ArrowDownwardIcon = createSvgIcon(
   'ArrowDownward',
 );
 
+export const FilterIcon = createSvgIcon(
+  <path d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z" />,
+  'FilterList',
```

Comment:
To match `FilterIcon`'s variable

```suggestion
  'Filter',
```